packages/gasket-plugin-mocha/index.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
         const path = require('path');
         const isReactProject = pkg.has('dependencies', 'react');
 
+        files.add(
+          path.join(__dirname, 'generator', 'test', 'setup.js')
```

Comment:
This is only actually needed for Next.js apps, since it utilizes `next/babel`, correct? Should we have:
```
const isNextProject = pkg.has('dependencies', 'next');
```
and add this setup file conditionally?